src/tasks/WasmAppBuilder/PInvokeTableGenerator.cs

```diff
@@ -443,15 +443,15 @@ private void EmitNativeToInterp(StreamWriter w, ref List<PInvokeCallback> callba
             pindex = 0;
             if (!is_void)
             {
-                sb.Append("&res");
+                sb.Append("(int*)&res");
```

Comment:
@BrzVlad Could you please review the change in this file?

Reply:
@maraf could you take a look

Reply:
LGTM, fwiw.  We generate `WasmInterpEntrySig_nnn` to have a signature like `void (*)(int* resultDest, int* arg0, ... int* argN, int* extraArg)`.  So casting all the addresses to `int*` is ok.